Dfe.Academies.Academisation.SubcutaneousTest/ProjectAggregate/LegacyProjectUpdateTests.cs

```diff
@@ -71,60 +59,9 @@ public async Task ProjectExists___FullProjectIsUpdated()
 		var updateResult = await legacyProjectController.Patch(updatedProject.Id, updatedProject);
 
 		// Assert
-		DfeAssert.OkObjectResult(updateResult);
-
-		var getResult = await legacyProjectController.Get(updatedProject.Id);
+		(_, var project) = DfeAssert.OkObjectResult(updateResult);	
 
-		(_, var getProject) = DfeAssert.OkObjectResult(getResult);
```

Comment:
Is it worth having another test (ProjectExists_FullProjectIsReturnedOnGet or something?) to throw the removed items here that are testing the GET request?

Not a massive amount of value in it but would be a way to repurpose these removed aspects..

Reply:
Sure, I'll add that now. Thanks for the comment.

Reply:
There is LegacyProjectGetTests.cs as well. But there's definitely know harm in adding an extra test here.

Reply:
Done.